lib/chunky_png/chunk.rb

```diff
@@ -368,13 +368,35 @@ def content
     # a translation of the keyword name. Finally, it supports bot compressed
```

Comment:
I think you (accidentally) updated the permissions of this file. Can you change it back to 644?